### PR TITLE
worked on functional favourite beers component

### PR DESCRIPTION
**Pull request by dwantula**

### CODE REVIEW

File: debug.log [hunk 1]
```diff
@@ -1,2 +1,4 @@
 [1119/110851.627:ERROR:directory_reader_win.cc(43)] FindFirstFile: System nie mo�e odnale�� okre�lonej �cie�ki. (0x3)
```
Comment (mi-chal) [inline debug.log]: Remove this file

File: src/shared/components/FavouriteButton/FavouriteButton.jsx [hunk 2]
```diff
@@ -1,21 +1,52 @@
...
+  //   const { id, img, name, description, ibu, abv } = beer;
+  //   // console.log(beer)
+  //   const newBeer = { id, name, description, img, ibu, abv };
+  //   const newBeers = [...beers, newBeer];
+  //   saveItemInLocalStorage('beers', newBeers);
+  // };
+
+
+
+
```
Comment (mi-chal) [inline src/shared/components/FavouriteButton/FavouriteButton.jsx]: I guess this should not be here.

File: src/services/beers.js [hunk 3]
```diff
@@ -12,8 +13,42 @@ export async function fetchRandomBeer() {
...
+        brewed
+      })
+    )
+  } catch (error) {
+    console.log(error)
+  }
+}
+
+export async function fetchBeers(params) {
+  const url = `${baseUrl}/beers?${params.filter(param => typeof param === 'string')}`
```
Comment (mi-chal) [inline src/services/beers.js]: This still doesn't work as it should (look at `Request URL` in `Headers` in Chrome Network tab).
Try with: 
`params.filter(param => !!param)}`

File: src/services/beers.js [hunk 4]
```diff
@@ -12,8 +13,42 @@ export async function fetchRandomBeer() {
...
+        ibu,
+        abv,
+        name,
+        ebc,
+        tagline,
+        img,
+        food,
+        brewed
+      })
+    )
```
Comment (mi-chal) [inline src/services/beers.js]: This logic repeats for `fetchBeersWithIds` and `fetchBeers`. Extract this to separate function.

File: debug.log (hunk 1 above)
Comment (dwantula) [reply debug.log]: i can't deleted this file i don't know why but i deleted all it is in this file

File: src/shared/components/FavouriteButton/FavouriteButton.jsx (hunk 2 above)
Comment (dwantula) [reply src/shared/components/FavouriteButton/FavouriteButton.jsx]: Done!

File: src/services/beers.js (hunk 4 above)
Comment (dwantula) [reply src/services/beers.js]: Done!

File: src/services/beers.js (hunk 3 above)
Comment (dwantula) [reply src/services/beers.js]: Done

**Review by mi-chal — APPROVED** [review]

**Pull request merged**